ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml

```diff
@@ -70,6 +70,13 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-ipi-preserve-bootstrap-ign
+- as: gcp-ipi-proxy-cco-manual-workload-identity-f28
```

Comment:
how about using `workload-identity` in the job name to replace `cco-manual-workload-identity`, we used `workload-identity` for other jobs like https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml#L2229